pkg/controller/lifecycle/actuator.go

```diff
@@ -629,6 +633,11 @@ func getShootResources(webhookCaBundle []byte, namespace, shootAccessServiceAcco
 		}}
 	)
 
+	isManagedSeed := shootNamespace == v1beta1constants.GardenNamespace
+	if isManagedSeed {
+		objectSelector = nil
```

Comment:
```suggestion
		objectSelector = metav1.LabelSelector{}
```

> An empty label selector matches all objects. A null label selector matches no objects.

ref: https://github.com/kubernetes/kubernetes/blob/9e2075b3c87061d25759b0ad112266f03601afd8/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L1239-L1240

Reply:
Thanks!